doc/faq/troubleshooting_faq.rst

```diff
@@ -34,31 +34,44 @@ and printing the ``__file__`` attribute::
 
 .. _locating-matplotlib-config-dir:
 
-:file:`.matplotlib` directory location
-======================================
+:file:`.matplotlib` and `.cache` directory location
+===================================================
 
 Each user has a matplotlib configuration directory which may contain a
 :ref:`matplotlibrc <customizing-with-matplotlibrc-files>` file. To
-locate your :file:`.matplotlib/` directory, use
+locate your :file:`matplotlib/` configuration directory, use
 :func:`matplotlib.get_configdir`::
 
     >>> import matplotlib as mpl
     >>> mpl.get_configdir()
-    '/home/darren/.matplotlib'
+    '/home/darren/.config/matplotlib'
 
 On unix-like systems, this directory is generally located in your
-:envvar:`HOME` directory.  On windows, it is in your documents and
-settings directory by default::
+:envvar:`HOME` directory under the :file:`.config/` directory. 
+
+In addition, users have a cache directory. On unix-like systems, this is 
+separate to the configuration directory by default. To locate your 
```

Comment:
I've seen both used, but don't have a particular preference, so will change (I think your suggestion is better here, really)